discovery/build.gradle

```diff
@@ -0,0 +1,6 @@
+
+dependencies {
+    compile files("${project.getProperty('jdk8')}/lib/tools.jar")
```

Comment:
Oh I think this can probably be removed. I totally missed it. It was likely added to the initial prototype to support dynamically attaching via the agent jar. Since that functionality was removed we shouldn't need to make tools.jar available.